external/src/main/java/com/redhat/cloud/policies/engine/history/AlertsHistoryEntry.java

```diff
@@ -0,0 +1,97 @@
+package com.redhat.cloud.policies.engine.history;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import java.util.Objects;
+import java.util.UUID;
+
+@Entity
+@Table(name = "alerts_history")
+public class AlertsHistoryEntry {
```

Comment:
Policies-ui-backend will need a similar class when reading from the database. One duplicated class isn't so bad but it would be even better if we could avoid duplicated code.